src/Groups/abelian_aut.jl

```diff
@@ -0,0 +1,89 @@
+function _isomorphic_gap_group(A::GrpAbFinGen; T=PcGroup)
+  # find independent generators
+  if isdiagonal(rels(A))
+    exponents = [Int(a) for a in diagonal(rels(A))]
+    A2 = A
+    A2_to_A = identity_map(A)
+    A_to_A2 = identity_map(A)
+    else
```

Comment:
Hmm, the indentation is funny here, the else seems to have the same indentation as the code before/after it?